container/test-app/iframe/microfrontend.html

```diff
@@ -102,9 +102,14 @@ <h1 id="title">Multi purpose demo page</h1>
       }
 
       function testGetContext() {
-        alert(
-          'LuigiClient.getContext()() called' + JSON.stringify(LuigiClient.getContext())
-        );
+        LuigiClient.sendCustomMessage({
+          id: 'my.contextMessage',
+          ...LuigiClient.getContext()
+        });
+        LuigiClient.uxManager().showAlert({
+          text: `LuigiClient.getContext() called ${JSON.stringify(this.ctx)}`,
+          type: 'info'
+        });
```

Comment:
I believe the sendCustomMessage for transmitting context back is enough. You can remove the showAlert